src/transactions/InvokeHostFunctionOpFrame.cpp

```diff
@@ -420,25 +420,24 @@ InvokeHostFunctionOpFrame::doApply(Application& app, AbstractLedgerTxn& ltx,
                     TTLEntry = ttlLtxe.current().data.ttl();
```

Comment:
nit: `TTLEntry` -> `ttlEntry` for consistency with the naming style

Reply:
Done